client/ayon_houdini/plugins/publish/save_scene.py

```diff
@@ -16,9 +17,10 @@ def process(self, context):
         # Filename must not have changed since collecting
         host = registered_host()
         current_file = host.get_current_workfile()
-        assert context.data['currentFile'] == current_file, (
-            "Collected filename from current scene name."
-        )
+        if context.data['currentFile'] != current_file:
+            raise PublishError(
+                message="Collected filename from current scene name."
```

Comment:
This error would be way clearer if it describes in the message what is the current scene file and what did it compare against.
```suggestion
                "Collected filename differs from current scene name."
```

Reply:
https://github.com/ynput/ayon-houdini/pull/190/commits/4502143ed6891c034bc74eed355f7796d6a0ef19